packages/product-download-page/style.module.css

```diff
@@ -1,7 +1,5 @@
 .root {
-  composes: .g-grid-container from global;
```

Comment:
This `composes` was not working, as it results in the markup `<div class=".g-grid-container">` rather than the expected `<div class="g-grid-container">`. I tried correcting this, but it seems like it's not actually what we want, so have instead dropped it.